components/authz-service/storage/postgres/datamigration/sql/01_v2_data_migrations.up.sql

```diff
@@ -8,12 +8,6 @@ UPDATE iam_members
     WHERE
         name = 'team:local:operators';
 
-UPDATE iam_statements
-    SET
-        role_id = role_db_id('editor')
-    WHERE
-        role_id = role_db_id('operator');
-
```

Comment:
These weren't doing anything -- we've introduced a `db_id` so we could rename them without having to update references. Before the strict `role_db_id` function, the WHERE would have selected `role_id=NULL`, matching nothing.